src/main/java/com/cloudbees/groovy/cps/tool/Translator.java

```diff
@@ -0,0 +1,814 @@
+            // Tree symbols created by the original JavacTask.parse() call to be thrown away,
+            // which breaks later processing.
+            // So for now, don't perform annotation processing
+            List<String> options = Arrays.asList("-proc:none");
+
+            Iterable<? extends JavaFileObject> files = fileManager.getJavaFileObjectsFromFiles(
+                    Collections.singleton(dgmJava));
+            JavaCompiler.CompilationTask task = javac1.getTask(null, fileManager, errorListener, options, null, files);
+            javac = (JavacTask) task;
+            trees = Trees.instance(javac);
+            elements = javac.getElements();
+            types = javac.getTypes();
+
+            Iterable<? extends CompilationUnitTree> parsed = javac.parse();
+            javac.analyze();
+
+            CompilationUnitTree dgmCut = getDefaultGroovyMethodCompilationUnitTree(parsed);
+
+            final DeclaredType closureType = types.getDeclaredType(elements.getTypeElement(Closure.class.getName()));
+
+            ClassSymbol dgm = (ClassSymbol) elements.getTypeElement("org.codehaus.groovy.runtime.DefaultGroovyMethods");
+            dgm.accept(new ElementScanner7<Void,Void>() {
+                public Void visitExecutable(ExecutableElement e, Void __) {
+                    if (isGdkMethodWithClosureArgument(e)
+                     && !EXCLUSIONS.contains(n(e))) {
+                        translate(dgmCut, e);
+                    }
+                    return null;
+                }
+
+                /**
+                 * Criteria:
+                 *      1. public static method
+                 *      2. has a Closure parameter in one of the arguments, not in the receiver
+                 */
+                private boolean isGdkMethodWithClosureArgument(ExecutableElement e) {
+                    return e.getKind() == ElementKind.METHOD
+                        && e.getModifiers().containsAll(PUBLIC_STATIC)
+                        && e.getParameters().subList(1, e.getParameters().size()).stream()
+                            .anyMatch(p -> types.isAssignable(p.asType(), closureType));
+                }
+            },null);
+
+            /*
+                private static MethodLocation loc(String methodName) {
+                    return new MethodLocation(CpsDefaultGroovyMethods.class,methodName);
+                }
+            */
+
+            JClass $MethodLocation = codeModel.ref("com.cloudbees.groovy.cps.MethodLocation");
+            $CpsDefaultGroovyMethods.method(JMod.PRIVATE|JMod.STATIC, $MethodLocation, "loc").tap( m -> {
+                JVar $methodName = m.param(String.class, "methodName");
+                m.body()._return(JExpr._new($MethodLocation).arg($CpsDefaultGroovyMethods.dotclass()).arg($methodName));
+            });
+        } finally {
+            if (fileManager!=null)
+                fileManager.close();
+        }
+
+        File dir = new File("out");
```

Comment:
Generally poor style; pass in `${project.build.directory}` or similar as a system property to `maven-exec-plugin` so that the output location is predictable, not sensitive to CWD.